Disable fallocate #562992 (only on hppa, alpha re-enables as per #661876)

This is only a workaround to fix FTBFS, the configure check is wrong and needs to be fixed:
http://www.amule.org/abugs/view.php?id=1572

--- a/src/ThreadTasks.cpp
+++ b/src/ThreadTasks.cpp
@@ -531,6 +531,15 @@ void CCompletionTask::OnExit()
 #include <stdlib.h>
 #include <errno.h>
 
+// #562992
+#if defined(__hppa__)
+#undef HAVE_FALLOCATE
+#endif
+
+#if defined(__hppa__)
+#undef HAVE_SYS_FALLOCATE
+#endif
+
 CAllocateFileTask::CAllocateFileTask(CPartFile *file, bool pause)
 	// GetPrintable is used to improve the readability of the log.
 	: CThreadTask(wxT("Allocating"), file->GetFullName().RemoveExt().GetPrintable(), ETP_High),
